typing/primitive.ml

```diff
@@ -85,7 +85,9 @@ let parse_declaration valdecl ~native_repr_args ~native_repr_res =
   let arity = List.length native_repr_args in
   let name, native_name, old_style_noalloc, old_style_float =
     match valdecl.pval_prim with
+    | name :: name2 :: "noalloc" :: "float" :: _
```

Comment:
What are these changes needed for?  I think these should go in a separate GPR.

Reply:
This fixes a bug in the handling of deprecated "noalloc" and "float" attributes (which is triggered by `raw_spacetime_lib.ml` as it exists on `trunk`), but indeed they are not needed in this PR; I will put it in a separate one.